kibana/Dockerfile

```
@@ -15,7 +15,7 @@ FROM centos:8 AS builder
 RUN cd /opt && \
```

```suggestion
RUN cd /tmp && \
```

To make sure the build will work, this is a backport of https://github.com/elastic/kibana/pull/120991/files which will be part of the future releases.

cc @jbudz

---

LGTM